nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/pom.xml

```diff
@@ -37,12 +37,12 @@
         <dependency>
 		    <groupId>org.apache.kafka</groupId>
 		    <artifactId>kafka-clients</artifactId>
-		    <version>0.8.2.2</version>
+		    <version>0.9.0.0</version>
 		</dependency>
         <dependency>
             <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka_2.9.1</artifactId>
-            <version>0.8.2.2</version>
+            <artifactId>kafka_2.10</artifactId>
```

Comment:
So Kafka 0.9.0.0 requires Kafka against Scala 2.10? That shouldn't interfere with anything else right?